filecoin-proofs/src/lib.rs

```diff
@@ -17,7 +17,6 @@ pub mod types;
 pub use self::api::*;
 pub use self::commitment_reader::*;
 pub use self::constants::SINGLE_PARTITION_PROOF_LEN;
-pub use self::pad_reader::*;
```

Comment:
Not exactly sure what the purpose of this line is, doesn't _seem_ to be needed, tests pass and I can use `fr32_reader::Fr32Reader` in an external library. Maybe I should have removed it in #1061.